CMakeLists.txt

```diff
@@ -82,6 +82,12 @@ if(CMAKE_C_FLAGS_RELEASE MATCHES "-O3")
   string(REPLACE "-O3" "-O2" CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE}")
 endif()
 
+# Enable link-time optimisations on release builds.
+check_c_compiler_flag(-flto HAS_FLTO_FLAG)
+if(HAS_FLTO_FLAG)
+  set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS} -flto")
```

Comment:
We should also enable it for RelWithDebInfo.

Based on [this commit](https://github.com/poelzi/ulatencyd/commit/32b7684ea919892ad63e1f1449acda7a2fed95b9#diff-af3b638bc2a3e6c650974192a53c7291R171) and other blurbs around the web, LTO needs the compiler flags to be _the same_ as the the linker flags.

CMAKE_EXE_LINKER_FLAGS_RELEASE
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO

If that's true then I wonder why it seems to have worked for @mhinz?